app/service/TaiService.scala

```diff
@@ -85,14 +85,8 @@ class TaiService @Inject()(taiConnector: TaiConnector,
         FRENoYears
       case freSeq if freSeq.forall(_.freAmount.isDefined) && freSeq.forall(_.freAmount.get.grossAmount == claimAmount) =>
         FREAllYearsAllAmountsSameAsClaimAmount
-      case freSeq if freSeq.forall(_.freAmount.isDefined) && freSeq.forall(_.freAmount.get.grossAmount != claimAmount) =>
-        FREAllYearsAllAmountsDifferent
-      case freSeq if freSeq.forall(_.freAmount.isDefined) && freSeq.exists(_.freAmount.get.grossAmount != claimAmount) =>
-        FREAllYearsSomeAmountsDifferent
-      case freSeq if freSeq.exists(_.freAmount.isEmpty) && freSeq.filterNot(_.freAmount.isEmpty).exists(_.freAmount.get.grossAmount > 0) =>
-        ComplexClaim
       case _ =>
-        TechnicalDifficulties
+        FRESomeYears
```

Comment:
What is the call to Tai fails, wont this return a successful FRESomeYears and allow them to continue with the journey?

Reply:
Actioned, please see changes